packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -2508,3 +2513,33 @@ class ToCsvButton extends StatelessWidget {
     );
   }
 }
+
+class RadioButton<T> extends StatelessWidget {
+  const RadioButton({
+    super.key,
+    required this.label,
+    required this.itemValue,
+    required this.groupValue,
+    this.onChanged,
+  });
+
+  final String label;
+  final T itemValue;
+  final T groupValue;
+  final void Function(T?)? onChanged;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        Radio<T>(
+          value: itemValue,
+          groupValue: groupValue,
+          materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
+          onChanged: onChanged,
+        ),
+        Text(label),
```

Comment:
This should likely be in an expanded to adapt to all layout sizes where this will be used. And give the Text an overflow style of ellipsis